latest/manifest.tmpl

```diff
@@ -1,16 +1,13 @@
-image: owncloud/ubuntu:latest
+image: owncloud/ubuntu:20.04
+tags:
+  - eoan
```

Comment:
Is tag eoan right? Shouldn't be focal?

Reply:
I cannot judge on this - who knows what this is about? @xoxys maybe? THX

Reply:
Right. 
- Ubuntu 19.10 = Eoan Ermine = tag "eoan"
- Ubuntu 20.04 LTS = Focal Fossa = tag "focal"

But I also dont know why `image: owncloud/ubuntu:latest` was changed... @jnweiger?

Reply:
latest is anyhow a bit strage - shouldn't latest be a simple link to what ever we define to be latest? Having an extra build on it's own code base fells wrong ....

Reply:
oh, yes eoan is an ubuntu code name. I'll change that manifest to say focal

Not sure what latest was. It was 18.04 for long time and nobody cared to update to 19.10, so maybe remove latest completely? 
If it is possible with a link or tag, then fine with me. I have no idea, why that was a separate build.